[2024-06-21T01:46:15.800+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-21T01:46:15.827+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: testing_dag.load manual__2024-06-21T01:45:50+00:00 [queued]>
[2024-06-21T01:46:15.836+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: testing_dag.load manual__2024-06-21T01:45:50+00:00 [queued]>
[2024-06-21T01:46:15.837+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-21T01:46:15.850+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): load> on 2024-06-21 01:45:50+00:00
[2024-06-21T01:46:15.859+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=397) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-21T01:46:15.862+0000] {standard_task_runner.py:63} INFO - Started process 399 to run task
[2024-06-21T01:46:15.862+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'testing_dag', 'load', 'manual__2024-06-21T01:45:50+00:00', '--job-id', '194', '--raw', '--subdir', 'DAGS_FOLDER/main_dag.py', '--cfg-path', '/tmp/tmp0i4ect07']
[2024-06-21T01:46:15.864+0000] {standard_task_runner.py:91} INFO - Job 194: Subtask load
[2024-06-21T01:46:15.911+0000] {task_command.py:426} INFO - Running <TaskInstance: testing_dag.load manual__2024-06-21T01:45:50+00:00 [running]> on host 39e4faefdcb7
[2024-06-21T01:46:16.000+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='***@example.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='testing_dag' AIRFLOW_CTX_TASK_ID='load' AIRFLOW_CTX_EXECUTION_DATE='2024-06-21T01:45:50+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-21T01:45:50+00:00'
[2024-06-21T01:46:16.001+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-21T01:46:16.041+0000] {logging_mixin.py:188} INFO - Equities,TJX,TJX Companies,Unnamed: 3,75.98,37468.4207623162,14761.023496753583,2846850.609520785,Applebead,2023-02-28,13990024665034623786,Unnamed: 11
[2024-06-21T01:46:16.095+0000] {logging_mixin.py:188} INFO - Connection successful.
[2024-06-21T01:46:16.100+0000] {logging_mixin.py:188} INFO - Error: syntax error at or near "Companies"
LINE 1: COPY raw_external_fund(Equities,TJX,TJX Companies,Unnamed: 3...
                                                ^
[2024-06-21T01:46:16.101+0000] {main_dag.py:135} ERROR - syntax error at or near "Companies"
LINE 1: COPY raw_external_fund(Equities,TJX,TJX Companies,Unnamed: 3...
                                                ^

[2024-06-21T01:46:16.103+0000] {python.py:237} INFO - Done. Returned value was: 1
[2024-06-21T01:46:16.103+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-21T01:46:16.135+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=testing_dag, task_id=load, run_id=manual__2024-06-21T01:45:50+00:00, execution_date=20240621T014550, start_date=20240621T014615, end_date=20240621T014616
[2024-06-21T01:46:16.198+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-21T01:46:16.232+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-21T01:46:16.236+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
